src/modules/TimeScale.js

```diff
@@ -632,29 +638,15 @@ class TimeScale {
     let yrCounter = 0
     let unit = 'minute'
 
-    let remainingSecs = 60 - firstVal.minSecond
-
+    let remainingSecs = 60 - currentSecond
     let firstTickPosition = remainingSecs * secondsWidthOnXAxis
-    let firstTickValue = firstVal.minMinute + 1
-    let minute = firstTickValue + 1
```

Comment:
* For the first tick minute and value were different because of this incrementation.
* Also if first minute is 59, it was incorrectly added as `{ value: 60, minute: 61 }`

Resolved both of these problems by moving addition of the first tick inside a loop below.